### PR TITLE
Hasan/fixed babel traverse

### PR DESCRIPTION
**Pull request by hasan-deriv**
It syncs with upstream master and fixed vulnerable versions of ip package

### CODE REVIEW

**Review by nada-deriv — APPROVED** [review]

File: .yarnrc.yml [hunk 1]
```diff
@@ -11,5 +11,3 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
-
-yarnPath: .yarn/releases/yarn-3.6.3.cjs
```
Comment (amam-deriv) [inline .yarnrc.yml:15]: why is this removed?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions/checkout@v4
       -
         name: GPG conf
-        uses: actions/github-script@v6
+        uses: actions/github-script@4020e461acd7a80762cdfff123a1fde368246fa4
```
Comment (amam-deriv) [inline .github/workflows/ci.yml:54]: why are we targeting specific branch instead of tag

File: .yarnrc.yml (hunk 1 above)
Comment (hasan-deriv) [reply .yarnrc.yml:15]: not sure. It's removed in the original repo. I just pulled it.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (hasan-deriv) [reply .github/workflows/ci.yml:54]: it's more secure.

**Review by amam-deriv — APPROVED** [review]
